src/Features/VisualBasic/Portable/ExtractInterface/VisualBasicExtractInterfaceService.vb

```diff
@@ -158,5 +159,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractInterface
 
             Return CreateFinalSolution(unformattedSolution, documentIds, docToRootMap)
         End Function
+
+        Friend Overrides Function TryGetLanguageSpecificErrorMessage(containingType As ISymbol, <NotNullWhen(True)> ByRef errorMessage As String) As Boolean
+
+            If TryCast(containingType, IModuleSymbol) Is Nothing Then
+                errorMessage = VBFeaturesResources.Could_not_extract_interface_colon_Cannot_extract_interface_for_modules
```

Comment:
slightly confused.  teh test says "if the containing is *not* a module".  shouldn't this be `is not nothing`? or `typeof containingtype is imodulesymbol`?

Reply:
hmm true, need to revisit why it passed as expected with manual testing. Maybe I was changing to be more succinct VB and forgot the `Not`